warehouse/templates/manage/account/publishing.html

```diff
@@ -27,6 +27,55 @@ <h1 class="page-title">{{ oidc_title() }}</h1>
 
   {{ oidc_desc() }}
 
+  <h2>{% trans %}Manage providers{% endtrans %}</h2>
+  
+  <h3>Projects with active providers</h3>
```

Comment:
Going to not be very clean, but we need to similarly show a "No publishers are currently configured." here instead of an empty table.

Reply:
@woodruffw Still need to update this!

Reply:
Yep, that's the last thing I think. Doing now...

Reply:
ce551cf6..1617165c